CHANGELOG.md

```diff
@@ -7,24 +7,41 @@ be found.
 
 ## 17.03.0-ce (2017-03-01)
 
+**IMPORTANT**: Starting with this release, Docker is on a monthly release cycle and uses a
+new YY.MM versioning scheme to reflect this. In addition to a monthly release, more stable
```

Comment:
I wonder if "more stable" is the right terminology here. Even as a maintainer, I don't quite understand this. Are we saying that the monthly releases are bleeding edge, and shouldn't be run by users that value stability, but the quarterly releases will be somehow better? Or do we mean to say that the quarterly releases will be supported for a longer period? (I actually don't know the answer here.)

How can users distinguish a "monthly" release from a "quarterly" release?